File: base/src/main/java/de/hipp/pnp/base/fivee/converter/persistent/Feature5eConverter.java
package de.hipp.pnp.base.fivee.converter.persistent;

import de.hipp.pnp.base.fivee.Feature5e;
import javax.persistence.Converter;
import org.springframework.stereotype.Component;

@Converter(autoApply = true)
@Component
public class Feature5eConverter extends BaseConverter<Feature5e> {

  public Feature5eConverter() {
    this.type = new Feature5e();
  }
}

File: src/main/kotlin/de/hipp/pnp/base/converter/persistent/Feature5eConverter.kt
package de.hipp.pnp.base.converter.persistent

import de.hipp.pnp.base.fivee.Feature5e
import jakarta.persistence.Converter
import org.springframework.stereotype.Component

@Converter(autoApply = true)
@Component
class Feature5eConverter : BaseConverter<Feature5e?>() {
    init {
        this.type = Feature5e()
    }
}
